test/parallel/test-console.js

```diff
@@ -76,17 +102,33 @@ assert.equal('foo bar\n', strings.shift());
 assert.equal('foo bar hop\n', strings.shift());
 assert.equal("{ slashes: '\\\\\\\\' }\n", strings.shift());
 assert.equal('inspect\n', strings.shift());
+assert.equal('foo\n', strings.shift());
+assert.equal('foo bar\n', strings.shift());
+assert.equal('foo bar hop\n', strings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", strings.shift());
+assert.equal('inspect\n', strings.shift());
+assert.equal('foo\n', errStrings.shift());
+assert.equal('foo bar\n', errStrings.shift());
+assert.equal('foo bar hop\n', errStrings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", errStrings.shift());
+assert.equal('inspect\n', errStrings.shift());
+assert.equal('foo\n', errStrings.shift());
+assert.equal('foo bar\n', errStrings.shift());
+assert.equal('foo bar hop\n', errStrings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", errStrings.shift());
+assert.equal('inspect\n', errStrings.shift());
 assert.equal("{ foo: 'bar', inspect: [Function] }\n", strings.shift());
 assert.equal("{ foo: 'bar', inspect: [Function] }\n", strings.shift());
 assert.notEqual(-1, strings.shift().indexOf('foo: [Object]'));
 assert.equal(-1, strings.shift().indexOf('baz'));
 assert.equal('Trace: This is a {"formatted":"trace"} 10 foo',
-             strings.shift().split('\n').shift());
```

Comment:
should this not be an addition instead of a change of the variable? It seems like we should still be testing `strings` as well as `errStrings`